std_msgs/Float64 angle
std_msgs/Float64[] angles
std_msgs/String[] cameras
geometry_msgs/Vector3 voffset
geometry_msgs/Vector3 vcurrent
geometry_msgs/Quaternion[] qoffset
geometry_msgs/Quaternion[] qcurrent